contracts/staking/contracts/src/stake/MixinStakeStorage.sol

```diff
@@ -93,9 +85,9 @@ contract MixinStakeStorage is
         // load from storage
         balance = balancePtr;
         // sync
-        uint256 currentEpoch = getCurrentEpoch();
-        if (currentEpoch > balance.currentEpoch) {
-            balance.currentEpoch = currentEpoch.downcastToUint32();
+        uint256 currentEpoch_ = currentEpoch;
```

Comment:
Oh, cool, I've been wanting to use this convention for local conflicts, but I wasn't sure other people were on board.